mm2src/coins/nft/storage/wasm/mod.rs

```diff
@@ -0,0 +1,59 @@
+use crate::nft::storage::NftStorageError;
+use mm2_db::indexed_db::{DbTransactionError, InitDbError};
+use mm2_err_handle::prelude::*;
+
+pub mod nft_idb;
+pub mod wasm_storage;
+
+pub type WasmNftCacheResult<T> = MmResult<T, WasmNftCacheError>;
+
+impl NftStorageError for WasmNftCacheError {}
+
+#[derive(Debug, Display)]
+pub enum WasmNftCacheError {
+    ErrorSerializing(String),
+    ErrorDeserializing(String),
+    ErrorSaving(String),
+    ErrorLoading(String),
+    ErrorClearing(String),
```

Comment:
I think `Error` should the suffix instead of prefix here e.g `LoadingError`..wdyt?

Reply:
Yeah, I understand what you mean. I just used similar enumeration names as in `DbTransactionError`. Its a common practice in our project when we deal with IDB as I can see.
![image](https://github.com/KomodoPlatform/atomicDEX-API/assets/52405288/b7894289-69f8-41b7-b0e7-2ed5597943e0)

Reply:
Ah okay cool